### DISCUSSION TITLE
just a noob question

### DISCUSSION THREAD

**Discussion by Aneeqasif** [rafaelmardojai/blanket#138] [Q&A] [answered]
sir i wanna know where the sound files are located on my system that blanket is using. I have one or two my own sound files that i like and i wanna replace the default ones. plz guide me where can i find the source directory o sounds so that i can replace them with my own one

**Comment by rafaelmardojai** [comment, marked as answer]
The sound files are installed as [GResources](https://developer.gnome.org/gio/stable/GResource.html), so they are combined into a binary bundle during the build.

You would need to download Blanket's source code, replace the files located at `data/resources/sounds`, and build it.

So nop, there's no easy way to do what you want.
